src/test/java/mServer/test/JsonFileReader.java

```diff
@@ -0,0 +1,32 @@
+package mServer.test;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.junit.Assert.fail;
+
+/**
+ * Reads a json file
+ */
+public class JsonFileReader {
+    
+    private JsonFileReader() {}
+    
+    public static JsonObject readJson(String filePath) {
+        try {
+            URI u = JsonFileReader.class.getResource(filePath).toURI();
+            Path path = Paths.get(u);
+            String jsonOutput = new String(Files.readAllBytes(path));
+            return new Gson().fromJson(jsonOutput, JsonObject.class);        
+        } catch(JsonSyntaxException | IOException | URISyntaxException ex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)